javascript-bank/account.js

```diff
@@ -0,0 +1,42 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (Number.isInteger(amount) && Math.sign(amount) === 1) {
+    this.transactions.push(new Transaction('deposit', amount));
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  if (Number.isInteger(amount) && Math.sign(amount) === 1) {
+    this.transactions.push(new Transaction('withdrawal', amount));
+    return true;
+  } else {
+    return false;
+  }
+};
+
+Account.prototype.getBalance = function () {
+  if (this.transactions.length === 0) {
+    return 0;
+  }
```

Comment:
If you declare `balance` outside of the `for` loop and give it a starting value of `0`, this conditional will no longer be necessary.